opentelemetry-otlp/src/exporter/http/logs.rs

```diff
@@ -14,14 +14,20 @@ impl LogExporter for OtlpHttpClient {
             .clone()
             .ok_or(OTelSdkError::AlreadyShutdown)?;
 
-        let (body, content_type) = self
+        let (body, content_type, content_encoding) = self
             .build_logs_export_body(batch)
             .map_err(OTelSdkError::InternalFailure)?;
 
-        let mut request = http::Request::builder()
+        let mut request_builder = http::Request::builder()
             .method(Method::POST)
             .uri(&self.collector_endpoint)
-            .header(CONTENT_TYPE, content_type)
+            .header(CONTENT_TYPE, content_type);
+
+        if let Some(encoding) = content_encoding {
+            request_builder = request_builder.header("Content-Encoding", encoding);
```

Comment:
nit: Move Content-Encoding to const string, or is it available from http crate already?